stdlib/seq.mli

```diff
@@ -150,7 +150,7 @@ val uncons : 'a t -> ('a * 'a t) option
 val length : 'a t -> int
 (** [length xs] is the length of the sequence [xs].
 
-    The sequence [xs] must be finite.
+    This function terminates only if the sequence [xs] is finite.
```

Comment:
If in the other cases you wrote

> May not terminate if [xs] is infinite.

here I would write

> Does not terminate if [xs] is infinite.

Reply:
Fixed.